functions/Add-DbaComputerCertificate.ps1

```diff
@@ -54,87 +54,87 @@ Add-DbaComputerCertificate -Path C:\temp\cert.cer
+    foreach ($computer in $ComputerName) {
 			
-			if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
-				$Password = ((65 .. 90) + (97 .. 122) | Get-Random -Count 29 | ForEach-Object { [char]$_ }) -join "" | ConvertTo-SecureString -AsPlainText -Force
-			}
+      if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
+        $Password = ((65 .. 90) + (97 .. 122) | Get-Random -Count 29 | ForEach-Object { [char]$_ }) -join "" | ConvertTo-SecureString -AsPlainText -Force
+      }
 			
-			$scriptblock = {
-				$Store = $args[2]
-				$Folder = $args[3]
+      $scriptBlock = {
+        $Store = $args[2]
+        $Folder = $args[3]
 				
-				$cert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2
-				$cert.Import($args[0], $args[1], [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::DefaultKeySet)
-				Write-Verbose "Importing cert to $Folder\$Store"
-				$tempstore = New-Object System.Security.Cryptography.X509Certificates.X509Store($Folder, $Store)
-				$tempstore.Open('ReadWrite')
-				$tempstore.Add($cert)
-				$tempstore.Close()
+        $cert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2
+        $cert.Import($args[0], $args[1], [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::DefaultKeySet)
+        Write-Verbose "Importing cert to $Folder\$Store"
+        $tempStore = New-Object System.Security.Cryptography.X509Certificates.X509Store($Folder, $Store)
+        $tempStore.Open('ReadWrite')
+        $tempStore.Add($cert)
+        $tempStore.Close()
 				
-				Write-Verbose "Searching Cert:\$Store\$Folder"
-				Get-ChildItem "Cert:\$store\$folder" -Recurse | Where-Object { $_.Thumbprint -eq $cert.Thumbprint }
-			}
+        Write-Verbose "Searching Cert:\$Store\$Folder"
+        Get-ChildItem "Cert:\$Store\$Folder" -Recurse | Where-Object { $_.Thumbprint -eq $cert.Thumbprint }
+      }
 			
-			if ($PScmdlet.ShouldProcess("local", "Connecting to $computer to import cert")) {
-				try {
-					Invoke-Command2 -ComputerName $computer -Credential $Credential -ArgumentList $certdata, $Password, $store, $folder -ScriptBlock $scriptblock -ErrorAction Stop |
-					Select-DefaultView -Property FriendlyName, DnsNameList, Thumbprint, NotBefore, NotAfter, Subject, Issuer
-				}
-				catch {
-					Stop-Function -Message $_ -ErrorRecord $_ -Target $computer -Continue
-				}
-			}
-		}
-	}
+      if ($PScmdlet.ShouldProcess("local", "Connecting to $computer to import cert")) {
+        try {
+          Invoke-Command2 -ComputerName $computer -Credential $Credential -ArgumentList $certdata, $Password, $Store, $Folder -ScriptBlock $scriptblock -ErrorAction Stop |
+            Select-DefaultView -Property FriendlyName, DnsNameList, Thumbprint, NotBefore, NotAfter, Subject, Issuer
+        }
+        catch {
+          Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $computer -Continue
```

Comment:
Hey @gbargsley - the connection errors are just for the top connecting part. When in doubt, you can leave that part off.